pkg/apis/internal.acorn.io/v1/memory.go

```diff
@@ -0,0 +1,33 @@
+package v1
+
+import (
+	"cuelang.org/go/cue/literal"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+func ParseMemoryMap(memory map[string]string) (map[string]int64, error) {
+	resources := map[string]int64{}
+	for service, value := range memory {
```

Comment:
even though its not user facing, id prefer you not use the term service. i just want to avoid the conflation

Reply:
Workload in place of service here as well?

Reply:
This isn't necessarily tied to a workload anymore since anything can call it to create the memory map. Updated it to just be `key`.